x-pack/solutions/security/plugins/security_solution/public/reports/components/ai_value/index.tsx

```diff
@@ -18,15 +18,28 @@ import { ExecutiveSummary } from './executive_summary';
 import { AlertProcessing } from './alert_processing';
 import { useValueMetrics } from '../../hooks/use_value_metrics';
 import { useKibana } from '../../../common/lib/kibana';
+import { useAIValueExportContext } from '../../providers/ai_value/export_provider';
 
 interface Props {
   setHasAttackDiscoveries: React.Dispatch<boolean>;
   from: string;
   to: string;
 }
 
-export const AIValueMetrics: React.FC<Props> = ({ setHasAttackDiscoveries, from, to }) => {
+export const AIValueMetrics: React.FC<Props> = (props) => {
+  const { setHasAttackDiscoveries } = props;
   const { uiSettings } = useKibana().services;
+  const exportContext = useAIValueExportContext();
+  const setReportInputForExportContext = exportContext?.setReportInput;
+
+  let from = props.from;
+  let to = props.to;
+
+  if (exportContext?.forwardedState) {
+    const { timeRange } = exportContext.forwardedState;
+    from = timeRange.from;
+    to = timeRange.to;
+  }
```

Comment:
```suggestion
  const { from, to } = useMemo(() => {
    if (exportContext?.forwardedState) {
      const { timeRange } = exportContext.forwardedState;
      return {
        from: timeRange.from,
        to: timeRange.to,
      };
    }
    return {
      from: props.from,
      to: props.to,
    };
  }, [props.from, props.to, exportContext?.forwardedState]);
```
 Fix the re-render issue by memoizing `from` and `to` with `useMemo`. Currently, they are reassigned on every render, creating new references and causing `useValueMetrics` to re-run unnecessarily.